src/components/Header.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import FlatButton from 'material-ui/FlatButton';
+import Divider from 'material-ui/Divider';
+
+import './styles/header.css';
+
+const Header = props => (
+  <header className="restrict-width flex">
+    <div id="menu" className="flex">
+      <div id="logo-container">
+        <img id="logo" src="/logo.png" alt="logo" />
+      </div>
+      <FlatButton
+        label="Home"
+        primary={props.selectedPage === 'home'}
+        onClick={() => props.changePage(props.selectedPage, 'home')}
+      />
+      <FlatButton
+        label="Send"
+        primary={props.selectedPage === 'send'}
+        onClick={() => props.changePage(props.selectedPage, 'send')}
+      />
+      <FlatButton
+        label="Track"
+        primary={props.selectedPage === 'track'}
+        onClick={() => props.changePage(props.selectedPage, 'track')}
+      />
+      <div style={{ flexGrow: 1 }} />
```

Comment:
@bill-feng Is there a reason to put inline style here instead of writing in css?